### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by vadd98** [jinay1991/spleeter#12]
Hi, I'd like to implement this library in a cross-platform app for Windows and Mac.
Could it be possible to build the source code for Windows as it is now, or it would require some work on the code?

**Comment by jinay1991** [comment]
Yes, it can work with Windows provided that all the dependencies are compiled for it and updated "third_party" directory to use correct tarballs.
